developers_chamber/qa/base.py

```diff
@@ -137,6 +158,11 @@ def _print_results(self):
             LOGGER.info(result)
 
     def run(self):
+        if not self._is_repo_clean():
+            raise ClickException(
+                'QA check requires repository to be clean, please remove any staged, unstaged or untracked files.'
+            )
```

Comment:
Some users were surprised that repo is automatically cleaned (using stash) between checks. :trollface: